Server/Communication/tasks.py

```diff
@@ -33,40 +34,106 @@ def is_message_expired(message):
 
 	return False
 
+def send_notification(device, message):
+	"""
+	Sends the given device a notification.
+
+	:type device: FCMDevice
+	:param message: The model of the user's phone
+
+	:type messsage: AsyncMessage
+	:param message: Contains the data about the notification settings
+	"""
+	LOGGER.debug("Sending the user a notification!")
```

Comment:
The 3 log messages here would be more useful if the device was included:
<device name>: Sending the user a notification.

In case many are sent at once, could see which ones failed and which succeeded.